### PR TITLE
[tflite] hardSwish support. Float only.

### PR DESCRIPTION
**Pull request by vuzelac-cadence**

### CODE REVIEW

File: lib/Importer/TFLiteModelLoader.cpp [hunk 1]
```diff
@@ -1233,6 +1236,9 @@ Error TFLiteModelLoader::loadUnaryArithmetic(const tflite::Operator *op,
   NodeValue output;
   if (opCode == tflite::BuiltinOperator_LOGISTIC) {
     output = F_->createSigmoid(opInfo.name, outTy, input);
+  } else if (opCode == tflite::BuiltinOperator_HARD_SWISH) {
+    return loadHardSwish(op, opInfo, input);
```
Comment (mciprian13) [inline lib/Importer/TFLiteModelLoader.cpp:1240]: Looks unnecessary to create the `loadHardSwish` function. You should remove that function and put here only:
```
output = F_->createHardSwish(opInfo.name, input);
```

File: lib/Importer/TFLiteModelLoader.cpp [hunk 2]
```diff
@@ -1233,6 +1236,9 @@ Error TFLiteModelLoader::loadUnaryArithmetic(const tflite::Operator *op,
...
+    return setOutputNodeValue(op, output);
```
Comment (mciprian13) [inline lib/Importer/TFLiteModelLoader.cpp:1241]: This line should be removed.

**Review by zrphercule — APPROVED** [review]
LGTM! It seems our CI is breaking, but a fix is on its way. Could you please rebase it later so that we can land it? Thanks!

**Comment by vuzelac-cadence** [comment]
@zrphercule this is rebased now

**Pull request closed without merging**